package.json

```diff
@@ -198,6 +198,7 @@
     "serve-static": "^1.13.1",
     "shell-quote": "1.6.1",
     "stacktrace-parser": "^0.1.3",
+    "uuid": "^3.0.1",
```

Comment:
Is this necessary? uuid has been in use without this dependency (see https://github.com/facebook/react-native/pull/11573/files#diff-d7db9e5e49c3ea163f89547a0be6a438L19 in this PR).

This is one of the blockers, as adding the uuid package may conflict with another uuid dependency that we use internally at Facebook.

Reply:
This looks kind of bad, it probably only works because of some transitive dependency that also includes uuid.

Not sure what the best solutions would be, we could vendor the uuid module used internally or maybe add it to fbjs.

Another solution that _could_ work is to use the uuid v4 import directly so `require('uuid/v4')` (https://github.com/kelektiv/node-uuid#quickstart---commonjs-recommended)

Reply:
Per #16543 looks like the missing `uuid` package is causing issues for some folks.